config/log4j.properties

```diff
@@ -85,3 +99,7 @@ log4j.additivity.state.change.logger=false
 log4j.logger.kafka.authorizer.logger=WARN, authorizerAppender
 log4j.additivity.kafka.authorizer.logger=false
 
+
+# turn this on to log metrics
+#log4j.logger.kafka.server.BrokerServerMetrics=INFO, metricsAppender, KAFKA
```

Comment:
The idea is that we'll tell customers "When you get support, uncomment these two lines"?